### ISSUE TITLE
Anyone tried to train this code with Imagenet from scratch ? 

### ISSUE THREAD

**Issue by KyleZheng1997** [lucidrains/vit-pytorch#55]

**Comment by lucidrains** [comment]
@KyleZheng1997 Hi Kyle, so there is one gotcha when training attention networks with Adam, and that is we exclude the parameters of the LayerNorm for weight decay. Or you can just turn off weight decay, it usually makes minor difference

Also try a smaller learning rate, perhaps starting with 3e-4 and working your way down with the scheduler
